persistence/in-memory-simple/src/main/java/com/slalom/example/db/UserRepositorySimpleAdapter.java

```diff
@@ -8,6 +8,7 @@
 import java.util.Map;
 import java.util.Optional;
 
+// todo : gros :+1: pour le mode in memory en pur java (sans spring) je pense que la sur couche spring Hazel Cast est overkill pour ce genre d'utilisation
```

Comment:
Oui en fait, je veux prouver qu'il est facil de changer l'implemention. A la base, je voulais faire un exemple in memory java simple + in memory DB (h2 ou mongo). Mais Java 10 est pas encore pret pour ce genre de truc, H2/JPA/Hibernate est pas vraiment encore compatible, donc le seul exemple simple que j'ai trouve c'est d'utiliser hazelcast

Reply:
yes bonne idée ;) 
c'est un peu ce que je te propose dans le fichier application/spring-app/src/main/java/com/slalom/example/spring/config/Config.java